ui/app/models/totp-key.js

```diff
@@ -10,30 +10,50 @@ import { withFormFields } from 'vault/decorators/model-form-fields';
 import { withModelValidations } from 'vault/decorators/model-validations';
 import { withExpandedAttributes } from 'vault/decorators/model-expanded-attributes';
 import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
+import { isPresent } from '@ember/utils';
 
 const ALGORITHMS = ['SHA1', 'SHA256', 'SHA512'];
 const DIGITS = [6, 8];
 const SKEW = [0, 1];
```

Comment:
Unless these are referenced in other places in this file, they can be use directly as the values for the `possibleValues` keys below